date/date.php

```diff
@@ -469,7 +469,7 @@ function mktime(
     ?int $day = null,
     ?int $year = null,
     #[Deprecated('Use the new timezone handling functions instead', since: '5.3')]
-    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $is_dst = -1
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '5.6')] $is_dst = -1
```

Comment:
Optional `$is_dst` was available only for a short period: https://3v4l.org/u4D30#v5523 - these are latest PHP 5.5 & 5.6 releases.
I'd change this to `(from: '5.5', to: '5.6')`.

Reply:
Right! Fixed it